Documentation/special-services/pod-mutator.md

```diff
@@ -0,0 +1,76 @@
+# PodMutator Service
+To provide pod mutation capabilities for finer-grained control over rkt containerization concepts and debugging needs, 
+this proposal introduces a new optional stage1 app/service with special semantics that is lightweight.
```

Comment:
I am not convinced that a service is a better way to go as well. But the problem for a series of command like `enter`, `gc` is that it will be somehow hard to control the resource usage of those commands. But with a service, we could probably enforce the resource quota of the service.